.github/workflows/app-cd.yml

```diff
@@ -6,8 +6,14 @@ permissions:
   contents: read  # This is required for actions/checkout
 
 # run on merge since you cannot push to main directly
+# on:
+#   push: 
+#     branches:
+#       - main
+#     paths:
+#       - front/**
 on:
-  push: 
+  pull_request:
```

Comment:
Why did this contribute to fixing the issue? It feels like it is vaguely the same thing as "on push".

Reply:
It doesn't it was to do with env vars again, 

this change was just to trigger the CD on this PR